### ISSUE TITLE
Expo support?

### ISSUE THREAD

**Issue by SebastianKurp** [aeirola/react-native-svg-asset-plugin#12]
Stumbled upon this project,
was wondering if it works with Expo.
 Followed the instructions, but haven't been able to get it to run in Expo yet.

**Comment by aeirola** [comment]
Hi, I haven't been planning on supporting Expo due to the fact that Expo already contains the [react-native-svg](https://github.com/react-native-community/react-native-svg) library which provides runtime SVG rendering. For Expo projects, I'd probably use something like [react-native-svg-transformer](https://github.com/kristerkari/react-native-svg-transformer) instead of this asset plugin.

I tried playing around in an Expo project, but couldn't find a way to get the asset plugin working. Might be related to how Expo manages [assets](https://docs.expo.io/versions/latest/guides/assets/) differently than pure React Native projects.

Nonetheless, I should probably add a mention to the readme file that Expo is currently not supported. Thanks for bringing this up!

**Comment by MrPumpking** [comment]
Hi, sorry for digging up an old thread but I thought that maybe someone would find this useful - I've managed to get this project working with Expo. I've set up my metro.config.js like so:

```js
// metro.config.js

const { getDefaultConfig } = require('@expo/metro-config')
const defaultConfig = getDefaultConfig(__dirname)

defaultConfig.transformer.assetPlugins.unshift('react-native-svg-asset-plugin')

module.exports = defaultConfig
```

So far it seems to be working pretty well. I've tested this using Expo 41 in both Expo Go app and standalone Android build. Haven't tested it on iOS though, will send and update when I get to it.
